xml/ha_hawk2_manage_i.xml

```diff
@@ -363,6 +363,9 @@
      &hawk2; uses the <command>crm_resource&nbsp;</command> <option>-U</option>
      command. The resource can move back to its original location or it may
      stay where it is (depending on resource stickiness).
+     <remark>taroth 2018-03-02: ygao, if I got it right from c#4 in fate#324508
+     this option is deprecated but as the formatting in Fate was broken, I could
+     not figure out with what to replace it - please advise</remark>
```

Comment:
The short option "-U" still remains, long options "--un-migrate" and "--un-move" have been replaced by "--clear".

So the description in here is basically fine. But we'd probably better use the term  "Clear" instead of "Unmigrate" in the GUI,  which would at least be consistent with crm_resource and crmsh.

Btw, we can also check the good formatted information from here:
https://wiki.clusterlabs.org/wiki/Pacemaker_2.0_Changes

The tool changes are in here:
https://wiki.clusterlabs.org/wiki/Pacemaker_2.0_Tool_Changes

Reply:
Thanks, did not know https://wiki.clusterlabs.org/wiki/Pacemaker_2.0_Tool_Changes so far. 
Regarding the UI: That's a very good point (especially as in a recent training I heard that the 'unmigrate' term often leads to confusion with customers). I will follow up with krig and ayoub on that and file a bug report for Hawk2, if necessary.

Reply:
Thank you, unmigrate is definitely not a very comfortable / intuitive term.